modules/python/test/test_misc.py

```diff
@@ -223,8 +223,8 @@ def test_parse_to_bool_convertible(self):
                              msg=get_conversion_error_msg(convertible_false, 'bool: false', actual))
 
     def test_parse_to_bool_not_convertible(self):
-        for not_convertible in (1.2, np.float(2.3), 's', 'str', (1, 2), [1, 2], complex(1, 1),
-                                complex(imag=2), complex(1.1), np.array([1, 0], dtype=np.bool)):
```

Comment:
The same for `float(2.3)`

Reply:
np.float(2.3) is float(2.3) and there is already 1.2, so I removed it.